
  0%| | 0/1 [00:0/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/site-packages/torch/nn/parallel/_functions.py:64: UserWarning: Was asked to gather along dimension 0, but all input tensors were scalars; will instead unsqueeze and return a vector.
  warnings.warn('Was asked to gather along dimension 0, but all '
100%|█| 1/1 [00:0[INFO|trainer.py:2107] 2025-02-19 09:17:41,806 >> ***** Running Evaluation *****
[INFO|trainer.py:2109] 2025-02-19 09:17:41,807 >>   Num examples = 133219
[INFO|trainer.py:2112] 2025-02-19 09:17:41,807 >>   Batch size = 16






















































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































100%|█| 8327/8327









100%|█| 10/10 [01:50<00:









100%|███████████████████████████████████████████████████████████| 10/10 [01:34<00:00,  9.51s/it]









100%|███████████████████████████████████████████████████████████| 10/10 [01:05<00:00,  6.53s/it]









100%|███████████████████████████████████████████████████████████| 10/10 [00:35<00:00,  3.53s/it]









100%|███████████████████████████████████████████████████████████| 10/10 [00:29<00:00,  3.02s/it]








100%|███████████████████████████████████████████████████████████| 10/10 [00:17<00:00,  1.70s/it]




100%|███████████████████████████████████████████████████████████| 10/10 [00:10<00:00,  1.01s/it]


 90%|██████████████████████████████████████████████████████      | 9/10 [00:05<00:00,  1.73it/s]
Attempted to log scalar metric eval_loss:
1.5274477005004883
Attempted to log scalar metric eval_random_top1:
0.5595906
Attempted to log scalar metric eval_recall@100:
0.984837
Attempted to log scalar metric eval_verifier_top1_accuracy@100:
0.38968916
Attempted to log scalar metric eval_voting_top1_accuracy@100:

[INFO|configuration_utils.py:351] 2025-02-19 11:02:17,103 >> Configuration saved in ../../execution/exec/checkpoint-1/config.json
[INFO|modeling_utils.py:889] 2025-02-19 11:02:22,232 >> Model weights saved in ../../execution/exec/checkpoint-1/pytorch_model.bin
Attempted to log scalar metric eval_weighted_voting_top1_accuracy@100:
0.78771797
Attempted to log scalar metric eval_recall@75:
0.9787718
Attempted to log scalar metric eval_verifier_top1_accuracy@75:
0.39840788
Attempted to log scalar metric eval_voting_top1_accuracy@75:
0.79234268
Attempted to log scalar metric eval_weighted_voting_top1_accuracy@75:
0.79006823
Attempted to log scalar metric eval_recall@50:
0.97103867
Attempted to log scalar metric eval_verifier_top1_accuracy@50:
0.4181956
Attempted to log scalar metric eval_voting_top1_accuracy@50:
0.78779378
Attempted to log scalar metric eval_weighted_voting_top1_accuracy@50:
0.78195603
Attempted to log scalar metric eval_recall@25:
0.94814253
Attempted to log scalar metric eval_verifier_top1_accuracy@25:
0.45178165
Attempted to log scalar metric eval_voting_top1_accuracy@25:
0.77839272
Attempted to log scalar metric eval_weighted_voting_top1_accuracy@25:
0.77164519
Attempted to log scalar metric eval_recall@20:
0.93851403
Attempted to log scalar metric eval_verifier_top1_accuracy@20:
0.4555724
Attempted to log scalar metric eval_voting_top1_accuracy@20:
0.76959818
Attempted to log scalar metric eval_weighted_voting_top1_accuracy@20:
0.76671721
Attempted to log scalar metric eval_recall@10:
0.89886277
Attempted to log scalar metric eval_verifier_top1_accuracy@10:
0.47975739
Attempted to log scalar metric eval_voting_top1_accuracy@10:
0.74374526
Attempted to log scalar metric eval_weighted_voting_top1_accuracy@10:
0.73502654
Attempted to log scalar metric eval_recall@5:
0.83874147
Attempted to log scalar metric eval_verifier_top1_accuracy@5:
0.51576952
Attempted to log scalar metric eval_voting_top1_accuracy@5:
0.69112964
Attempted to log scalar metric eval_weighted_voting_top1_accuracy@5:
0.67983321
Attempted to log scalar metric eval_recall@2:
0.70295679
Attempted to log scalar metric eval_verifier_top1_accuracy@2:
0.54829416
Attempted to log scalar metric eval_voting_top1_accuracy@2:
0.56118271
Attempted to log scalar metric eval_weighted_voting_top1_accuracy@2:
0.55299469
Attempted to log scalar metric eval_runtime:
6275.1441
Attempted to log scalar metric eval_samples_per_second:
21.23
Attempted to log scalar metric epoch:
1.0
{'eval_loss': 1.5274477005004883, 'eval_random_top1': 0.5595906, 'eval_recall@100': 0.984837, 'eval_verifier_top1_accuracy@100': 0.38968916, 'eval_voting_top1_accuracy@100': 0.79302502, 'eval_weighted_voting_top1_accuracy@100': 0.78771797, 'eval_recall@75': 0.9787718, 'eval_verifier_top1_accuracy@75': 0.39840788, 'eval_voting_top1_accuracy@75': 0.79234268, 'eval_weighted_voting_top1_accuracy@75': 0.79006823, 'eval_recall@50': 0.97103867, 'eval_verifier_top1_accuracy@50': 0.4181956, 'eval_voting_top1_accuracy@50': 0.78779378, 'eval_weighted_voting_top1_accuracy@50': 0.78195603, 'eval_recall@25': 0.94814253, 'eval_verifier_top1_accuracy@25': 0.45178165, 'eval_voting_top1_accuracy@25': 0.77839272, 'eval_weighted_voting_top1_accuracy@25': 0.77164519, 'eval_recall@20': 0.93851403, 'eval_verifier_top1_accuracy@20': 0.4555724, 'eval_voting_top1_accuracy@20': 0.76959818, 'eval_weighted_voting_top1_accuracy@20': 0.76671721, 'eval_recall@10': 0.89886277, 'eval_verifier_top1_accuracy@10': 0.47975739, 'eval_voting_top1_accuracy@10': 0.74374526, 'eval_weighted_voting_top1_accuracy@10': 0.73502654, 'eval_recall@5': 0.83874147, 'eval_verifier_top1_accuracy@5': 0.51576952, 'eval_voting_top1_accuracy@5': 0.69112964, 'eval_weighted_voting_top1_accuracy@5': 0.67983321, 'eval_recall@2': 0.70295679, 'eval_verifier_top1_accuracy@2': 0.54829416, 'eval_voting_top1_accuracy@2': 0.56118271, 'eval_weighted_voting_top1_accuracy@2': 0.55299469, 'eval_runtime': 6275.1441, 'eval_samples_per_second': 21.23, 'epoch': 1.0}
Attempted to log scalar metric train_runtime:
6302.0045
Attempted to log scalar metric train_samples_per_second:
0.0
Attempted to log scalar metric total_flos:
0
[INFO|trainer.py:1352] 2025-02-19 11:02:31,072 >>
Training completed. Do not forget to share your model on huggingface.co/models =)
100%|█| 1/1 [1:44
02/19/2025 11:04:53 - INFO - __main__ - *** Evaluate ***
[INFO|trainer.py:2107] 2025-02-19 11:04:57,915 >> ***** Running Evaluation *****
[INFO|trainer.py:2109] 2025-02-19 11:04:57,915 >>   Num examples = 133219
[INFO|trainer.py:2112] 2025-02-19 11:04:57,915 >>   Batch size = 16
/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/site-packages/torch/nn/parallel/_functions.py:64: UserWarning: Was asked to gather along dimension 0, but all input tensors were scalars; will instead unsqueeze and return a vector.
  warnings.warn('Was asked to gather along dimension 0, but all '


























































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































 38%|███████████████████▍                               | 3166/8327 [37:44<1:00:19,  1.43it/s]Traceback (most recent call last):
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/mila/x/xut/.vscode-server/extensions/ms-python.debugpy-2025.0.1-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/__main__.py", line 71, in <module>
    cli.main()
  File "/home/mila/x/xut/.vscode-server/extensions/ms-python.debugpy-2025.0.1-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/../debugpy/server/cli.py", line 501, in main
    run()
  File "/home/mila/x/xut/.vscode-server/extensions/ms-python.debugpy-2025.0.1-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/../debugpy/server/cli.py", line 351, in run_file
    runpy.run_path(target, run_name="__main__")
  File "/home/mila/x/xut/.vscode-server/extensions/ms-python.debugpy-2025.0.1-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 310, in run_path
    return _run_module_code(code, init_globals, run_name, pkg_name=pkg_name, script_name=fname)
  File "/home/mila/x/xut/.vscode-server/extensions/ms-python.debugpy-2025.0.1-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 127, in _run_module_code
    _run_code(code, mod_globals, init_globals, mod_name, mod_spec, pkg_name, script_name)
  File "/home/mila/x/xut/.vscode-server/extensions/ms-python.debugpy-2025.0.1-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 118, in _run_code
    exec(code, run_globals)
  File "/home/mila/x/xut/github/DIVERSE/DIVERSE/code/src/run_ner.py", line 461, in <module>
    main()
  File "/home/mila/x/xut/github/DIVERSE/DIVERSE/code/src/run_ner.py", line 410, in main
    result = trainer.evaluate()
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/site-packages/transformers/trainer.py", line 1993, in evaluate
    output = eval_loop(
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/site-packages/transformers/trainer.py", line 2146, in evaluation_loop
    loss, logits, labels = self.prediction_step(model, inputs, prediction_loss_only, ignore_keys=ignore_keys)
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/site-packages/transformers/trainer.py", line 2348, in prediction_step
    loss, outputs = self.compute_loss(model, inputs, return_outputs=True)
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/site-packages/transformers/trainer.py", line 1766, in compute_loss
    outputs = model(**inputs)
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/site-packages/torch/nn/parallel/data_parallel.py", line 161, in forward
    outputs = self.parallel_apply(replicas, inputs, kwargs)
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/site-packages/torch/nn/parallel/data_parallel.py", line 171, in parallel_apply
    return parallel_apply(replicas, inputs, kwargs, self.device_ids[:len(replicas)])
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/site-packages/torch/nn/parallel/parallel_apply.py", line 78, in parallel_apply
    thread.join()
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/threading.py", line 1011, in join
    self._wait_for_tstate_lock()
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/threading.py", line 1027, in _wait_for_tstate_lock
    elif lock.acquire(block, timeout):
KeyboardInterrupt
Traceback (most recent call last):
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/mila/x/xut/.vscode-server/extensions/ms-python.debugpy-2025.0.1-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/__main__.py", line 71, in <module>
    cli.main()
  File "/home/mila/x/xut/.vscode-server/extensions/ms-python.debugpy-2025.0.1-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/../debugpy/server/cli.py", line 501, in main
    run()
  File "/home/mila/x/xut/.vscode-server/extensions/ms-python.debugpy-2025.0.1-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/../debugpy/server/cli.py", line 351, in run_file
    runpy.run_path(target, run_name="__main__")
  File "/home/mila/x/xut/.vscode-server/extensions/ms-python.debugpy-2025.0.1-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 310, in run_path
    return _run_module_code(code, init_globals, run_name, pkg_name=pkg_name, script_name=fname)
  File "/home/mila/x/xut/.vscode-server/extensions/ms-python.debugpy-2025.0.1-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 127, in _run_module_code
    _run_code(code, mod_globals, init_globals, mod_name, mod_spec, pkg_name, script_name)
  File "/home/mila/x/xut/.vscode-server/extensions/ms-python.debugpy-2025.0.1-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 118, in _run_code
    exec(code, run_globals)
  File "/home/mila/x/xut/github/DIVERSE/DIVERSE/code/src/run_ner.py", line 461, in <module>
    main()
  File "/home/mila/x/xut/github/DIVERSE/DIVERSE/code/src/run_ner.py", line 410, in main
    result = trainer.evaluate()
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/site-packages/transformers/trainer.py", line 1993, in evaluate
    output = eval_loop(
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/site-packages/transformers/trainer.py", line 2146, in evaluation_loop
    loss, logits, labels = self.prediction_step(model, inputs, prediction_loss_only, ignore_keys=ignore_keys)
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/site-packages/transformers/trainer.py", line 2348, in prediction_step
    loss, outputs = self.compute_loss(model, inputs, return_outputs=True)
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/site-packages/transformers/trainer.py", line 1766, in compute_loss
    outputs = model(**inputs)
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/site-packages/torch/nn/parallel/data_parallel.py", line 161, in forward
    outputs = self.parallel_apply(replicas, inputs, kwargs)
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/site-packages/torch/nn/parallel/data_parallel.py", line 171, in parallel_apply
    return parallel_apply(replicas, inputs, kwargs, self.device_ids[:len(replicas)])
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/site-packages/torch/nn/parallel/parallel_apply.py", line 78, in parallel_apply
    thread.join()
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/threading.py", line 1011, in join
    self._wait_for_tstate_lock()
  File "/home/mila/x/xut/.conda/envs/verifier_env/lib/python3.8/threading.py", line 1027, in _wait_for_tstate_lock
    elif lock.acquire(block, timeout):
